### DISCUSSION TITLE
can't see the plots

### DISCUSSION THREAD

**Discussion by Idrake07** [Chia-Network/chia-blockchain#6549] [Support]
Please tell me, I sowed the plots on an external hard drive, at the end he went to sleep, the plots are displayed on the hard drive, but nothing has changed in chia blockchain (he does not see them), I checked the plots in chia plot status on them 1 error  someone encountered, it makes sense to wait until the plots  turn on or have to be overwritten?
![error](https://user-images.githubusercontent.com/85481614/120979953-c16ec180-c77e-11eb-8b7c-6bded3160747.jpeg)

**Comment by BrandtH22** [comment]
@Idrake07 , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
